Doc/library/traceback.rst

```diff
@@ -346,7 +346,7 @@ capture data for later printing in a lightweight fashion.
       some containing internal newlines. :func:`~traceback.print_exception`
       is a wrapper around this method which just prints the lines to a file.
 
-   .. method::  format_exception_only(*, show_group=False)
+   .. method::  format_exception_only(*, show_group=False, limit_group=True)
```

Comment:
I don't think we need a new parameter. Treat this as a bug and make the limit apply always.

Reply:
Oups, this is a typo now :)